library/alloc/src/raw_vec.rs

```diff
@@ -401,7 +401,8 @@ impl<T, A: Allocator> RawVec<T, A> {
 
     fn capacity_from_bytes(excess: usize) -> usize {
         debug_assert_ne!(mem::size_of::<T>(), 0);
```

Comment:
We could use `size_of_item` in `debug_assert_ne!` to simplify.

Reply:
I kept this because I was not sure of how to

Reply:
Ah okay, let's keep it then.